docs/source/examples/pec_tutorial.myst

```diff
@@ -0,0 +1,310 @@
+
+```{code-cell} ipython3
+def get_cnot_representation(edge: Tuple[int, int]) -> pec.OperationRepresentation:
+    cnot_circuit = cirq.Circuit(cirq.CNOT(cirq.LineQubit(edge[0]), cirq.LineQubit(edge[1])))              
+    
+    rep_exact_prob = 1 - np.sqrt(1 - cnot_error_prob)
+    return pec.represent_operation_with_local_depolarizing_noise(
+        cnot_circuit, 
+        noise_level=rep_exact_prob,
+    )
+
+
+def get_representations(computer: nx.Graph) -> List[pec.OperationRepresentation]:
+    return [get_cnot_representation(edge) for edge in computer.edges]
+```
+
+### Helper methods
+
+It will be useful later to us to have a number of utility functions for counting
+CNOT gates and operations.
+
+```{code-cell} ipython
+def get_num_cnot_count(circuit: Union[cirq.Circuit, qiskit.QuantumCircuit]) -> int:
+    """Determine number of cnot gates in a given `Circuit` object."""    
+    # Count CNOT gates for `cirq`-type circuit objects:
+    num_cnots: int = 0
+    for instruction in circuit.instructions:
+        if isinstance(instruction.operator, gates.CNot):
+            num_cnots += 1
+    return num_cnots
+
+def get_oneq_count(circuit: Union[cirq.Circuit, qiskit.QuantumCircuit]) -> int:
+    return len(circuit.instructions) - get_num_cnot_count(circuit)
+```
+
+
+## Define the executor
+
+Now that we have a circuit, we define the `execute` function which inputs a circuit and returns an expectation value - here, the frequency of sampling the correct bitstring.
+
+```{code-cell} ipython3
+def execute(
+    circuits: Union[cirq.Circuit, List[cirq.Circuit]], 
+    backend,
+    shots: int,
+    correct_bitstring: str,
+    is_noisy: bool = True,
+) -> List[float]:
+    """Executes the input circuit(s) and returns ⟨A⟩, where 
+    A = |correct_bitstring⟩⟨correct_bitstring| for each circuit.
+    """
+    if not isinstance(circuits, list):
+        circuits = [circuits]
+    
+    # Store all circuits to run in list to be returned.
+    to_run: Union[List[braket.Circuit], List[qiskit.QuantumCircuit]] = []
+    
+    # Compile circuits to appropriate gateset.
+    for circuit in circuits:
+        circuit_to_run = circuit.copy()            
+        if is_noisy:            
+            circuit_to_run.apply_gate_noise(
+                Noise.Depolarizing(probability=cnot_error_prob/2), target_gates=gates.CNot
+            )
+        to_run.append(circuit_to_run)
+        
+    # Process each job.
+    results: List[float] = []
+    for circ_to_run in to_run:
+        job_result = backend.run(circ_to_run, shots=shots).result()
+        results.append(job_result.measurement_probabilities.get("".join(map(str, correct_bitstring)), 0.0))
+
+    return results
+```
+
+## Running the experiment
+
+PEC samples many circuits according to the input representations. For each unitary, $U$ in the circuit with representation
+
+$$
+U = \sum_i a_i G_i,
+$$
+where channel $G_i$ replaces $U$ with (normalized) probability $|a_i|$.
```

Comment:
```suggestion
is replaced by $G_i$ with probability $|a_i| /\gamma$, where $\gamma=\sum_i |a_i|$.
```
This procedure is repeated multiple times, generating $M$ (`num_samples`) circuits. The associated noisy expectation values are re-combined by Mitiq according to the following formula

$$
\frac{\gamma}{M} \sum_{i=1}^M \sigma_i E_i ,
$$

which is an unbiased estimator of the true (noiseless) expectation value.